.github/actions/build_vgg_resnet_action/action.yml

```diff
@@ -0,0 +1,75 @@
+name: Setup and build networks; VGG16, Resnet50
+description: Build and run VV16 and Resnet50 with oneAPI base toolkit and oneAPI-Construction-Kit
+
+inputs:
+  workspace:
+    description: 'Main workflow workspace'
+    default: ${{ github.workspace }}
+
+runs:
+  # We don't want a new docker just a list of steps, so mark as composite
+  using: "composite"
+  steps:
+    - name: setup python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
```

Comment:
What's the reason for this specific version? Asking because current Python is 3.12; if there are problems with newer versions, it would be good to know about them.

Reply:
I think when I was initially setting up the github ci for tartan, one of the components needed were supported by 3.8 version. I'll have a run with 3.12 to confirm everything works fine and then update this separately.